openbas-front/src/admin/components/components/teams/Teams.tsx

```diff
@@ -94,8 +96,8 @@ const Teams = () => {
   const headers = [
     { field: 'team_name', label: 'Name', isSortable: true },
     { field: 'team_description', label: 'Description', isSortable: true },
```

Comment:
sortable: false?

Reply:
I add this on back-end
@Queryable(sortable = true)
    @Column(name = "team_description")
    @JsonProperty("team_description")
    private String description;

Reply:
sorry, I said for team_name

Reply:
Oh I rollback your changes.. sorry

Reply:
Fix